### PR TITLE
Add feature store spark integration test case

### PR DESCRIPTION
**Pull request by can-sun**
*Issue #, if available:*
None

*Description of changes:*
Add feature store spark integration test case

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: spark/processing/3.0/py3/docker/py37/Dockerfile.cpu [hunk 1]
```diff
@@ -108,6 +108,6 @@ WORKDIR $SPARK_HOME
 
 # Install the sagemaker feature store spark connector
 # https://docs.aws.amazon.com/sagemaker/latest/dg/batch-ingestion-spark-connector-setup.html
-RUN /usr/bin/python3 -m pip install sagemaker-feature-store-pyspark-3.0 --no-binary :all:
+RUN /usr/bin/python3 -m pip install sagemaker-feature-store-pyspark-3.0==1.1.2 --no-binary :all:
```
Comment (xiaoxshe) [inline spark/processing/3.0/py3/docker/py37/Dockerfile.cpu:111]: Can you link the PySpark version for feature store?  is 1.12 compatible for all the Spark Images from 3.0 all the way to 3.3?

File: test/resources/code/python/feature_store_py_spark/py_spark_feature_store_ingestion.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
Comment (xiaoxshe) [inline test/resources/code/python/feature_store_py_spark/py_spark_feature_store_ingestion.py:1]: Can you add a test for `CreateProcessingJob` to make sure Processing Job doesn't get impacted?

Comment (can-sun) [reply test/resources/code/python/feature_store_py_spark/py_spark_feature_store_ingestion.py:1]: The processing job is created via the `PySparkProcessor` which will create processing job with the image to be released and run. To be more specific, this `py_spark_feature_store_ingestion.py` will be executed by `test_sagemaker_feature_store_ingestion_multinode`

File: spark/processing/3.0/py3/docker/py37/Dockerfile.cpu (hunk 1 above)
Comment (can-sun) [reply spark/processing/3.0/py3/docker/py37/Dockerfile.cpu:111]: Yes, we support the Spark versions from 3.0 ~ 3.3

https://pypi.org/project/sagemaker-feature-store-pyspark-3.0/
https://pypi.org/project/sagemaker-feature-store-pyspark-3.1/
https://pypi.org/project/sagemaker-feature-store-pyspark-3.2/
https://pypi.org/project/sagemaker-feature-store-pyspark-3.3/

**Review by xiaoxshe — APPROVED** [review]

File: spark/processing/3.0/py3/docker/py37/Dockerfile.cpu (hunk 1 above)
Comment (can-sun) [reply spark/processing/3.0/py3/docker/py37/Dockerfile.cpu:111]: And maven repo is here:

https://mvnrepository.com/artifact/software.amazon.sagemaker.featurestore/sagemaker-feature-store-spark-sdk_2.12/1.1.2

**Review by mahendruajay — DISMISSED** [review]

**Review by mahendruajay — APPROVED** [review]

**Review by mahendruajay — APPROVED** [review]

**Pull request merged**